src/main/groovy/com/palantir/gradle/docker/DockerComposeTemplatePlugin.groovy

```diff
@@ -0,0 +1,54 @@
+package com.palantir.gradle.docker
+
+import com.google.common.base.Preconditions
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.tasks.Copy
+
+class DockerComposeTemplatePlugin implements Plugin<Project> {
+    @Override
+    void apply(Project project) {
+        DockerComposeTemplateExtension ext =
+            project.extensions.create('dockerComposeTemplate', DockerComposeTemplateExtension, project)
```

Comment:
I think this probably ought to be `dockerCompose` instead of `dockerComposeTemplate`